modules/ckanext-ytp_main/ckanext/ytp/schemas/organization.json

```diff
@@ -0,0 +1,127 @@
+{
+  "scheming_version": 1,
+  "organization_type": "organization",
+  "about_url": "http://github.com/ckan/ckanext-scheming",
+  "fields": [
+    {
+      "field_name": "title_translated",
+      "label": "Title",
+      "preset": "fluent_core_title_translated",
+      "form_languages": [ "fi", "sv", "en" ],
```

Comment:
Do these need `validators": "fluent_text",` validator?

Reply:
https://github.com/vrk-kpa/opendata/blob/dev/modules/ckanext-ytp_main/ckanext/ytp/schemas/presets.json#L132

The preset already sets that validator.